L1Trigger/L1TNtuples/plugins/L1CaloTowerTreeProducer.cc

```diff
@@ -99,6 +100,7 @@ L1CaloTowerTreeProducer::L1CaloTowerTreeProducer(const edm::ParameterSet& iConfi
   ecalToken_ = consumes<EcalTrigPrimDigiCollection>(iConfig.getUntrackedParameter<edm::InputTag>("ecalToken"));
   hcalToken_ = consumes<HcalTrigPrimDigiCollection>(iConfig.getUntrackedParameter<edm::InputTag>("hcalToken"));
   l1TowerToken_ = consumes<l1t::CaloTowerBxCollection>(iConfig.getUntrackedParameter<edm::InputTag>("l1TowerToken"));
+  decoderToken_ = esConsumes<CaloTPGTranscoder, CaloTPGRecord>();
```

Comment:
Certainly all these tokens, but I'd say also most (if not all) class parameters can be made `const` and initialized in the class initializer list